features/step_definitions/api/metrics_steps.rb

```diff
@@ -59,6 +59,16 @@ def map_or_unmap_metric(metric, mapped)
   assert_equal '', find_mapped_cell_in_table(name).text
 end
 
+Then /^I should (not )?see "([^"]*)" on the (metrics|methods) tab$/ do |negate, metric_name, tab_name|
```

Comment:
`negate` is unused, the step only checks for metric presence.

Reply:
Good catch :eyes: Let me review what I wanted to achieve here.

Reply:
Fixed. I did not remove `negate`, but added the logic that uses it. The current test does not use it with negation, but I think it might be useful for other tests.